pocket-dark.user.css

```diff
@@ -237,11 +238,11 @@
     border-right: 0 solid #e8e6e6 !important;
   }
   .queue_list .item, .grid .queue_list .item {
-    border-color: #000;
+    border-color: black;
```

Comment:
I think using :root vars is better here also

Reply:
The normal keyword seemed to make sense in this case. The `--black` variable is already  `#111`. I can rename it to `--almost-black` or make `--true-black: #000;` instead.

Reply:
The darkest should be #111 and lightest #eee really so we can add other shades to account for that and use them as such.

I presume the input fields need to be always darker.